.github/workflows/test.yml

```diff
@@ -0,0 +1,24 @@
+name: test
+
+on:
+  - push
+
+jobs:
+  cypress-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: isbang/compose-action@v1.0.0
+        with:
+          down-flags: '--volumes'
+      - name: Run tests
+        run: |
+          cd tests
+          npm install
+          npx cypress run
+      - name: Upload screenshots
+        if: failure()
+        uses: actions/upload-artifact@v3
+        with:
+          name: screenshots
+          path: tests/cypress/screenshots/
```

Comment:
This is quite cool - when the tests fail, Cypress spits out a screenshot of the page at the time of failure. With this, you can download those screenshots with the github UI.

Reply:
Really cool that we can see the context around the failure too! ❇️